### PR TITLE
PHP 8.2 Initial commit

### PR DESCRIPTION
**Pull request by Luke-Roy-IBM**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,19 +67,21 @@ wsk action update myAction myAction.php --kind php:8.0
 ### Local development
 
 ```
+./gradlew core:php8.2Action:distDocker
 ./gradlew core:php8.1Action:distDocker
 ./gradlew core:php8.0Action:distDocker
 ```
-This will produce the images `whisk/action-php-v8.1` and `whisk/action-php-v8.0` respectively.
+This will produce the images `whisk/action-php-v8.1` and `whisk/action-php-v8.1`, `whisk/action-php-v8.0` respectively.
```
Comment (akrabat) [inline README.md:74]: Should be: 

> `whisk/action-php-v8.2`, `whisk/action-php-v8.1` and `whisk/action-php-v8.0`

File: core/php8.2Action/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - mbstring
+    - mysqli
+    - pdo_mysql
+    - pdo_pgsql
+    - pdo_sqlite
+    - soap
+    - zip
+    - mongo
+- Added: Composer packages:
+    - [guzzlehttp/guzzle](https://packagist.org/packages/guzzlehttp/guzzle): 7.4.5
```
Comment (akrabat) [inline core/php8.2Action/CHANGELOG.md:37]: We should ship Guzzle version 7.7.0.

File: core/php8.2Action/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    - [ramsey/uuid](https://packagist.org/packages/ramsey/uuid): 4.4.0
```
Comment (akrabat) [inline core/php8.2Action/CHANGELOG.md:38]: We should ship ramsey/uuid version 4.7.4

File: core/php8.2Action/composer.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "config": {
+        "platform": {
+            "php": "8.2"
+        }
+    },
+    "require": {
+        "guzzlehttp/guzzle": "7.4.5",
+        "ramsey/uuid": "4.4.0"
```
Comment (akrabat) [inline core/php8.2Action/composer.json:9]: Should update to `4.7.4`

File: core/php8.2Action/composer.json [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{
+    "config": {
+        "platform": {
+            "php": "8.2"
+        }
+    },
+    "require": {
+        "guzzlehttp/guzzle": "7.4.5",
```
Comment (akrabat) [inline core/php8.2Action/composer.json:8]: Should update to `7.7.0`

**Review by akrabat — CHANGES_REQUESTED** [review]
Looks good. I think we should update the PHP packages to the latest versions though.

File: core/php8.2Action/composer.json (hunk 5 above)
Comment (Luke-Roy-IBM) [reply core/php8.2Action/composer.json]: Good Idea I implemented the suggestions

**Comment by Luke-Roy-IBM** [comment]
@akrabat  Good Idea I implemented your suggestions

**Review by akrabat — APPROVED** [review]
LGTM

**Pull request merged**